src/views/dialogs/TriggersDialog.tsx

```diff
@@ -33,10 +35,15 @@ export const TriggersDialog = ({
   const stringGetter = useStringGetter();
   const dispatch = useDispatch();
 
+  const onCloseDialog = () => {
+    abacusStateManager.clearTriggerOrdersInputValues();
```

Comment:
if doing this in the cleanup useEffect of the triggers form is the same, can we do it there instead? (so it's similar to other forms / inputs)